packages/matter.js/test/cluster/definitions/WindowCoveringClusterTest.ts

```diff
@@ -4,21 +4,29 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Crypto } from "../../../src/crypto/Crypto.js";
-Crypto.get = () =>
-    ({
-        // make Random data deterministic
-        getRandomData: (length: number) => {
-            return new Uint8Array(length);
-        },
-    }) as Crypto;
-
 import { WindowCovering, WindowCoveringCluster } from "../../../src/cluster/definitions/WindowCoveringCluster.js";
 import { ClusterServer } from "../../../src/cluster/server/ClusterServer.js";
+import { Crypto } from "../../../src/crypto/Crypto.js";
 import { BitFlags } from "../../../src/schema/BitmapSchema.js";
 
 describe("WindowCoveringCluster", () => {
     const WindowCovering_LF_PALF = WindowCoveringCluster.with("Lift", "PositionAwareLift");
+    let actualCrypto = Crypto.get;
```

Comment:
Why this needs to move here? Is not enoigh mocked?

Reply:
Left in accidentally, I think I moved it here initially before I realized I needed to edit a bunch of files and moved it to the test harness